### ISSUE TITLE
Add a DatabricksSQLCLI task

### ISSUE THREAD

**Issue by loicmathieu** [kestra-io/plugin-databricks#2]
### Feature description

See https://docs.databricks.com/dev-tools/databricks-sql-cli.html

**Comment by Malaydewangan09** [comment]
@loicmathieu 
Just to confirm, we are referring to the [Databricks SQL CLI](https://docs.databricks.com/aws/en/dev-tools/databricks-sql-cli), correct?
Not the [Databricks CLI](https://github.com/databricks/cli)?

**Comment by loicmathieu** [comment]
Maybe we should provide both.

@Ben8t WDYT?

**Comment by Ben8t** [comment]
Yes I think both are important. The "real" [Databricks CLI](https://docs.databricks.com/aws/en/dev-tools/cli/) is even more IMO:

> Access Databricks functionality using the Databricks command-line interface (CLI). The CLI wraps the Databricks REST API, so instead of sending REST API calls directly using curl or Postman, you can use the Databricks CLI to interact with Databricks.


@Malaydewangan09 I see you already started [PR](https://github.com/kestra-io/plugin-databricks/pull/112) on this. Can you make sure having a clear separation between the two CLI so Kestra user don't mix things together please?

(Looking quickly at the code I would have `io.kestra.plugin.databricks.cli.DatabricksCLI` and `io.kestra.plugin.databricks.cli.DatabricksSQLCLI` to match with Databricks semantic - but up for @loicmathieu reco here)
